src/app/pages/component-sidenav/component-sidenav.scss

```diff
@@ -1,88 +1,133 @@
-  width: $sidenav-width;
-  bottom: 0;
   overflow: auto;
-  height: 100%;
+}
 
-  &.mat-sidenav-opened {
-    box-shadow: 3px 0 6px rgba(0, 0, 0, .24);
-  }
+.docs-component-sidenav-inner-content {
+  display: flex;
+  flex-direction: row;
 
-  // Section divider
-  h3 {
-    border: none;
-    font-size: 10px;
-    letter-spacing: 1px;
-    line-height: $sidenav-spacing-unit * 3;
-    text-transform: uppercase;
-    font-weight: 400;
-    margin: 0;
-    padding: 0 ($sidenav-spacing-unit * 2);
+  // The rule will match the element following the router-outlet which will be the routed component.
+  router-outlet + * {
+    flex-grow: 1;
   }
+}
 
-  ul {
-    list-style-type: none;
-    margin: 0;
-    padding: 0;
+.mat-drawer {
+  &::-webkit-scrollbar {
+    height: 4px;
+    width: 4px;
   }
+}
 
-  // Sidenav navigation item
-  li {
-    border-bottom-width: 1px;
-    border-bottom-style: solid;
-    margin: 0;
-    padding: 0;
+.docs-component-viewer-nav {
+  position: sticky;
+  top: 25px;
 
+  .docs-component-viewer-nav-inner {
+    margin: 25px;
+    width: $sidenav-width;
+    max-height: 75vh;
+    overflow: auto;
 
-    // Hide the border on the last item
-    &:last-child {
-      border-color: transparent;
+    &::-webkit-scrollbar {
+      height: 4px;
+      width: 4px;
     }
 
-    > a {
-      box-sizing: border-box;
-      display: block;
-      font-size: 14px;
-      font-weight: 400;
-      line-height: ($sidenav-spacing-unit * 6) - 1;
-      text-decoration: none;
-      transition: all .3s;
-      padding: 0 ($sidenav-spacing-unit * 2);
+    h3 {
+      padding: 10px 15px;
+      font-weight: 700;
+      line-height: 16px;
+      margin: 0;
+      font-size: 13px;
+      cursor: pointer;
       position: relative;
 
-      &.docs-component-viewer-sidenav-item-selected {
-        font-weight: 600;
+      .mat-icon {
+        position: absolute;
+        right: 5px;
+        font-size: 18px;
       }
     }
+
+    hr {
+      padding: 0;
+      margin: 0;
+    }
+
+    ul {
+      list-style-type: none;
+      margin: -5px 0 5px 0;
+      padding: 0;
+      overflow: hidden;
+    }
+
+    li {
+      font-size: 13px;
+      line-height: 16px;
+      margin: 0;
+      padding: 5px 15px 5px 20px;
+    }
+
+    a {
+      text-decoration: none;
+    }
   }
 }
 
 .docs-component-sidenav-content {
-  min-height: 100%;
   display: flex;
   flex-direction: column;
+}
 
-  // The rule will match the element following the router-outlet which will be the routed component.
-  router-outlet + * {
-    flex-grow: 1;
+// Add specific rule to prevent default rule conflict
+.docs-component-viewer-sidenav-container.mat-drawer-container{
+  display: block;
+  overflow: hidden;
+  position: absolute;
+  top: 50px;
+  bottom: 0;
+  left: 0;
+  right: 0;
+}
+
+.docs-component-sidenav-inner-content {
+  display: flex; 
+  flex-direction: column;
+}
+
+.docs-component-sidenav-body-content {
+  display: flex;
+  min-height: calc(100vh - 264px);
```

Comment:
What is 264? It looks based on `$sidenav-width` somehow.

Reply:
264 is the footer + header nav height.

Reply:
Oh yeah lol of course the min-height doesn't have anything to do with sidenav width. Maybe add it as a variable with a comment?

Reply:
Comment for sure at the least